app/views/contexts/thank_you_context.py

```diff
@@ -19,5 +26,14 @@ def build_default_thank_you_context(session_data):
     return context
 
 
-def build_census_thank_you_context(display_address, census_type):
-    return {"display_address": display_address, "census_type": census_type}
+def build_census_thank_you_context(
+    session_data: SessionData, census_type_code: str, schema_theme: str
+) -> Mapping:
+    redirect_url = get_census_base_url(schema_theme, session_data.language_code)
+
+    return {
+        "display_address": session_data.display_address,
+        "census_type": census_type_code,
+        "hide_signout_button": False,
+        "sign_out_url": url_for("session.get_sign_out", redirect_url=redirect_url),
```

Comment:
The `redirect_url` parameter isn't sanitised. If you replace the url with something else, then we will send a redirect there e.g. http://localhost:5000/sign-out?redirect_url=https://bbc.co.uk works 😬 .